src/app/pages/OpenWalletPage/Features/FromLedger/__tests__/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { render, act, screen } from '@testing-library/react'
+import { FromLedger } from '..'
+import { MemoryRouter } from 'react-router-dom'
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}))
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => jest.fn(),
+}))
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => jest.fn(),
+}))
+
+jest.mock('../../../../../lib/ledger', () => ({
+  ...jest.requireActual('../../../../../lib/ledger'),
+  // Throws BLE not supported
+  canAccessBle: () => jest.fn().mockReturnValue(false),
+}))
+
+const renderComponent = () =>
+  render(
+    <MemoryRouter>
+      <FromLedger />
+    </MemoryRouter>,
+  )
+
+describe('<FromLedger />', () => {
+  it('should render component', async () => {
+    const { container } = renderComponent()
+
+    await act(() => Promise.resolve())
```

Comment:
Promise.resolve is useless here, but in general you don't want to use act in this case. If you want to test async changes use await waitFor(() => expect(...)